google/cloud/pubsub/samples/samples.cc

```diff
@@ -272,6 +294,24 @@ void DeleteSubscription(google::cloud::pubsub::SubscriptionAdminClient client,
   (std::move(client), argv.at(0), argv.at(1));
 }
 
+void ModifyPushConfig(google::cloud::pubsub::SubscriptionAdminClient client,
+                      std::vector<std::string> const& argv) {
+  //! [START pubsub_update_push_subscription] [modify-push-config]
```

Comment:
I presume "update" -v- "modify" was someone else's mistake, yes?

Reply:
Yes, as far as I can tell the other languages use `pubsub_update_push_subscription` as the region tag for the `ModifyPushConfig()` example :man_shrugging: